README.md

```diff
@@ -245,6 +245,36 @@ For a full list of all configuration options, see the `ActionCable::Server::Conf
 
 Also note that your server must provide at least the same number of database connections as you have workers. The default worker pool is set to 100, so that means you have to make at least that available. You can change that in `config/database.yml` through the `pool` attribute.
 
+## Security
+
+Websockets default implementation is vulnerable to [Cross-Site WebSocket Hijacking](https://www.christian-schneider.net/CrossSiteWebSocketHijacking.html)
```

Comment:
Can we write a bit about what the hijacking is, why we need protection, and maybe why it's safe to reuse the controller tokens?